src/MicroSDC.cpp

```diff
@@ -247,6 +253,16 @@ void MicroSDC::updateState(const std::shared_ptr<BICEPS::PM::NumericMetricState>
   notifyEpisodicMetricReport(newState);
 }
 
+void MicroSDC::updateState(const std::shared_ptr<BICEPS::PM::SetValueOperationState>& state)
+{
+  std::lock_guard<std::mutex> lock(runningMutex_);
+  if (!running_)
+  {
+    return;
+  }
+  auto newState = updateMdib(state);
```

Comment:
This should in future be notified by a episodic metric report, correct?
Leave a comment here

Reply:
Since we are just updating the operation state but not it's target state at this point, I would suggest using a `EpisodicOperationalStateReport` after a quick look at the standard.
The episodic metric report could be sent after the operation was successfully executed.